src/CoreWCF.Primitives/src/CoreWCF/Description/CustomAttributeProvider.cs

```diff
@@ -329,22 +352,22 @@ private static TProp GetProperty<TProp>(object obj, string propName)
             }
         }
 
-        public bool IsDefined(Type attributeType, bool inherit)
-        {
-            switch (ProviderType)
-            {
-                case AttributeProviderType.Type:
-                    return Type.GetTypeInfo().IsDefined(attributeType, inherit);
-                case AttributeProviderType.MethodInfo:
-                    return MethodInfo.IsDefined(attributeType, inherit);
-                case AttributeProviderType.MemberInfo:
-                    return MemberInfo.IsDefined(attributeType, inherit);
-                case AttributeProviderType.ParameterInfo:
-                    return ParameterInfo.IsDefined(attributeType, inherit);
-            }
-            Contract.Assert(false, "This should never execute.");
-            throw new InvalidOperationException();
-        }
+        //public bool IsDefined(Type attributeType, bool inherit)
```

Comment:
Can you delete this code instead of commenting it out.

Reply:
Nevermind, I just committed a change to your branch to delete that code and it's ready to merge.

Reply:
thanks!